packages/wrangler/src/__tests__/dev.test.ts

```diff
@@ -948,45 +946,113 @@ describe.sequential("wrangler dev", () => {
 		});
 
 		describe(".env", () => {
+			const processEnv = process.env;
+			beforeEach(() => (process.env = { ...processEnv }));
+			afterEach(() => (process.env = processEnv));
```

Comment:
The `.env` handling may mutate process.env so we make a copy for the duration of each test.